### PR TITLE
adding portfolio desktop version

### PR DESCRIPTION
**Pull request by Nejjari001**
setup the portfolio version desktop

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -394,7 +398,7 @@ li.frameworks-about p,
 li.skills-about p {
   padding-top: 6px;
   position: relative;
-  width: 327px;
+  width: auto;
```
Comment (CollinsTatang) [inline styles.css:401]: - [ ] Please kindly work on the design to match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848)
**Your Design**
you
**Figmal Design**
skill_Yours

File: styles.css [hunk 2]
```diff
@@ -425,14 +429,14 @@ span.icone-about {
...
-  width: 375px;
+  width: auto;
   background-color: #6070ff;
   padding: 114px 24px;
   border-top-left-radius: 75px;
 }
 
 .contact-content {
-  width: 295px;
+  width: auto;
```
Comment (CollinsTatang) [inline styles.css:442]: - [ ] Please kindly work on the background design to match that of [figmal](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848)
**Your Design**
yourStyle
**Figmal Design**
figmal_contact

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Nejjari001 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- 
## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.

- [ ] Please update your readme file and the [descriptive summary](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) of your pull request.

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Nejjari001, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights ✅

- Adherence to GitHub flow
- Implement suggested changes
- Good commit messages


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**